src/views/RecIM/components/Forms/ActivityForm/index.jsx

```diff
@@ -215,14 +243,19 @@ const ActivityForm = ({
   //re spreads fetched data to map to drop-down's once data has been loaded
   useEffect(() => {
     setNewInfo(currentInfo);
+    if (currentInfo.Logo !== null) {
+      setCropperImageData(currentInfo.Logo);
+    } else {
+      setCropperImageData(null);
+    }
```

Comment:
isn't it already null in this instance? is there a reason you have to set it to null again?